tests/test_lambdas/test_broker.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from unittest.mock import MagicMock
+
+from pytest import LogCaptureFixture
+
+from lbz.lambdas.broker import LambdaBroker
+
+
+class TestEventBroker:
+    def test_broker_works_properly(self) -> None:
+        func_1 = MagicMock()
+        mapper = {"x": func_1}
+        event = {"op": "x", "data": {"y": 1}}
+
+        LambdaBroker(mapper, event).handle()  # type: ignore
```

Comment:
You should also verify the returned value, so I guess you don't have to work with mocks in these tests 😉